docs/whats-new/production-release-notes.md

```diff
@@ -22,7 +22,7 @@ Frontend version: 1.0.0-20240306.1552 | Backend version: 1.95.14-20240311.1204
 
 | Type | Change| Description|
  ------ | ------- | ----------- |
-| **Feature** | Expose SAML features | Tenants can now manage SAML apps on the portal. Refer to [SAML Apps](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/saml) and [applications](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/applications) for more details. |
+| **Feature** | Expose SAML features | Tenants can now manage SAML apps on the portal. Refer to [SAML Apps](https://stg.docs.developer.tech.gov.sg/docs/techpass-tenant-guide/concepts/saml) and [applications](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/applications) for more details. |
```

Comment:
sorry! I just realised recently saml apps is only viewable, not manageable, can we change to 'can now view' (change in both stg & prod release notes)

Reply:
should we call out that cannot manage?